filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobFileSystemManager.scala

```diff
@@ -252,14 +234,7 @@ case class NativeBlobSasTokenGenerator(container: BlobContainerName, endpoint: E
     * @return an AzureSasCredential for accessing a blob container
     */
   def generateBlobSasToken: Try[AzureSasCredential] = for {
-    uri <- BlobPathBuilder.parseURI(endpoint.value)
-    configuredAccount <- BlobPathBuilder.parseStorageAccount(uri)
-    azureAccount <- findAzureStorageAccount(configuredAccount)
-    keys = azureAccount.getKeys.asScala
-    key <- keys.headOption.fold[Try[StorageAccountKey]](Failure(new Exception("Storage account has no keys")))(Success(_))
-    first = key.value
-    sskc = new StorageSharedKeyCredential(configuredAccount.value, first)
-    bcc = buildBlobContainerClient(sskc, endpoint, container)
+    bcc <- AzureUtils.buildContainerClientFromLocalEnvironment(container.toString, endpoint.toString, azureSubscription)
```

Comment:
Rather than creating the new `val azureSubscription` above, I would just:
```
... endpoint.toString, subscription.map(_.toString))
```

Reply:
Thanks! I like it.